Automattic/woocommerce-payments
# Fix SEPA transaction page

Fixes #1472 

Blocked until #1468 is merged.

#### Changes proposed in this Pull Request
Payment detail page expected certain properties to exist which don't with the SEPA payment method. 
Similar changes will be needed for Giropay and others. 

#### Testing instructions
Place an order with the SEPA payment method.
Open the order in wp-admin and click the payment transaction id to see load the transaction detail page. 
Page should render and look like:
![image](https://user-images.githubusercontent.com/1534605/114457324-218c2180-9b9b-11eb-9a59-f6ffeb2dba88.png)




*

-------------------
